book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.book = self
+    rental.person.add_rentals(rentals) unless rental.person.rentals.include?(rental)
+  end
```

Comment:
- [ ] Please `the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.` the method should get the person and the data and then it should create a new rental object with the date ,person and self which is the book info.